### ISSUE TITLE
Uncaught TypeError: Object #<Hoodie> has no method 'rejectWith' 

### ISSUE THREAD

**Issue by varunnaik** [hoodiehq/hoodie#265]
With Hoodie 0.7.0, the following code:

```
var hoodie = new Hoodie();
var promise = hoodie.account.signUp('test','test');
promise.then(
        function(p) {console.log("Success", p)},
        function(p) {console.log("Failure", p)}
)
```

fails with 

```
Uncaught TypeError: Object #<Hoodie> has no method 'rejectWith'
```

if the user could not be created for any reason. This does not happen with Hoodie 0.5.4. The error occurs in hoodie.js, line 2024.

**Comment by gr2m** [comment]
Please update to 0.7.1

**Comment by varunnaik** [comment]
Yup, it's solved with 0.7.1. Thanks :-)

**Comment by peetersn** [comment]
Awesome response time @gr2m, thanks!
